lib/taurus/qt/qtcore/util/emitter.py

```diff
@@ -175,6 +200,13 @@ def __init__(self, parent=None, name='', queue=None, method=None, cursor=None, s
             Qt.Qt.WaitCursor) if cursor is True else cursor
         self._cursor = False
         self.timewait = sleep
+        self.polling = polling
+        self.loopwait = int(loopwait*1e-3)
```

Comment:
Are you sure this is ok?
The default value of loopwait is 5. With this cast, the default value is going to be 0. Maybe you wanted to multiply by 1e3?